.gitignore

```diff
@@ -2,18 +2,9 @@ node_modules
 node_webkit_builds
 
 ._*
+.idea
 *.log
 .DS_Store
 
 builds
 shares
-.idea/.name
-.idea/encodings.xml
-.idea/jsLibraryMappings.xml
-.idea/libraries/starjunk_most_node_modules.xml
-.idea/misc.xml
-.idea/starjunk most.iml
-.idea/vcs.xml
-.idea/modules.xml
-.idea/workspace.xml
-.idea/watcherTasks.xml
```

Comment:
I didn't realize WebStorm added all this muck. One of the downfalls of using an IDE over a text editor.